docs/contributing.rst

```diff
@@ -71,15 +71,17 @@ Running the compiler tests
 ==========================
 
 The ``./scripts/tests.sh`` script executes most Solidity tests and
-runs ``aleth`` automatically if it is in the path, but does not download it,
+runs ``aleth`` automatically if it is in the path. The script does not download it,
 so you need to install it first. Please read on for the details.
 
-Solidity includes different types of tests, most of them bundled into the ``soltest``
-application. Some of them require the ``aleth`` client in testing mode, others require ``libz3``.
+Solidity includes different types of tests, most of them bundled into the `C++ Boost Test Framework <https://www.boost.org/doc/libs/1_69_0/libs/test/doc/html/index.html>`_ application ``soltest``.
+Some of them require the ``aleth`` client in testing mode, others require ``libz3``.
 
 To run a basic set of tests that require neither ``aleth`` nor ``libz3``, run
-``./scripts/soltest.sh --no-ipc --no-smt``. This script runs ``./build/test/soltest``
-internally.
+``./scripts/soltest.sh --no-ipc --no-smt``. This script runs ``./${BUILD_DIR}/test/soltest``
```

Comment:
Do people really want to know about this complication here?

Reply:
I don't understand what you mean. Remove the sentence that starts "This script runs..."?

Reply:
I meant, do the want to know about the fact that it actually runs something that uses `BUILD_DIR` from your environment. Perhaps it would be simpler to just talk about the default setting first and then tell them how to modify it, if they want.

Reply:
I've removed mention of what goes on underneath. This simplifes things. Now that there is a `--help`, that should be run for more detail.